Assets/MRTK/SDK/Experimental/Drawstring/Scripts/ElasticHandleLogic.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Physics.Experimental
+{
+    /// <summary>
+    /// Implements a two-handle elastic "stretch" logic, which allows for
+    /// either one or two pointers to stretch along a particular axis.
+    /// 
+    /// Built around the differential equations for a damped harmonic oscillator.
+    /// Intended for use with critically and over-damped oscillators.
+    /// 
+    /// Usage:
+    /// When a manipulation starts, call Setup.
+    /// Call Update() to compute an iteration of the system and obtain the calculated
+    /// distance between the handles.
+    /// </summary>
+    internal class ElasticHandleLogic
+    {
+        private Vector3 leftInitialPosition = Vector3.zero;
+        private Vector3 rightInitialPosition = Vector3.zero;
+
+        private LinearElasticSystem elasticSystem = null;
+
+        private bool isSetup;
+
+        /// <summary>
+        /// Initialize system with source info from controllers/hands.
+        /// Left and right handles must both be specified, but this does not
+        /// preclude single-hand interaction when Update() is called.
+        /// </summary>
+        /// <param name="leftHandleStart">World position of "left" handle point.</param>
+        /// <param name="rightHandleStart">World position of "right" handle point.</param>
+        /// <param name="extentInfo">Properties of the linear 1-D elastic extent to be manipulated</param>
+        /// <param name="elasticProperties">Properties of the elastic material/spring.</param>
```

Comment:
Can you comment more about the distinction between extendInfo and elasticProperties? If only because one of them is called ElasticExtendProperties and another is called ElasticProperties. By comment, I think I just mean discuss/elaborate on here (i.e. in this discussion)